### DISCUSSION TITLE
1.9.16

### DISCUSSION THREAD

**Discussion by T8RIN** [T8RIN/ImageToolbox#162] [Announcements]
## What's Changed
* Translations update
* Added PhotoPicker backport
* Fixed some minor issues
* Added color editing by text in color dialog
* Some icons changed

**Full Changelog**: https://github.com/T8RIN/ImageResizer/compare/1.9.15...1.9.16

<hr /><em>This discussion was created from the release <a href='https://github.com/T8RIN/ImageResizer/releases/tag/1.9.16'>1.9.16</a>.</em>

**Comment by J43git** [comment]
I wish I could code. This app is great! But if I knew how to code, I'd implement making presets by text too. The slider is all over the place for adding presets to the point where it becomes a guessing game.

**Comment by T8RIN** [comment]
Good suggestion, you can create an issue, so i would definitely won't forget about it :)

**Comment by T8RIN** [comment]
Added this in a new version

**Reply by J43git** [reply]
Thank you! Oh, by the way, I'm impressed with how much of a fast worker you are! I like that you implement stuff and get it out, rather than save it for big updates.

**Reply by T8RIN** [reply]
Thanks! I just love that :)
